src/common/output-stream.ts

```diff
@@ -58,9 +59,22 @@ export class OutputStream {
   }
 }
 
+export async function decodeRecordingsResponse(
```

Comment:
I am not a fan of changing parameter and not specifically returning result.  This is just odd and the function API does not match its name - `decode` should return decoded stuff.

Reply:
I agree, I'll rewrite it a bit.